fdbclient/admin_client.go

```diff
@@ -41,7 +41,8 @@ import (
 )
 
 const (
-	fdbcliStr = "fdbcli"
+	fdbcliStr                        = "fdbcli"
+	maintenanceModeDurationInSeconds = 600
```

Comment:
Shouldn't this be configurable?

Reply:
Yes. I will put this behind a flag and also make the duration configurable.